chat/src/main/java/org/navgurukul/chat/ChatBaseActivity.kt

```diff
@@ -35,6 +40,13 @@ open class ChatBaseActivity: AppCompatActivity() {
         }
     }
 
+    companion object {
+
+        fun newIntent(context: Context): Intent {
+            return Intent(context,ChatBaseActivity::class.java)
```

Comment:
It doesn't make sense to open ChatBaseActivity because it is just a baseclass, it will be an empty screen if we try to open that. We should open chat list screen that is on homescreen. We will need to make a decision on how the UX would look like as chat list screen is one index below the python screen in the stack

Reply:
This feature is actually we are not using mentorHelp button 's visibility is gone, this is already hidden till we take a decision on this. If required we can even remove this code.